File: Resources/TOI-Zero_68/A1/040/040.c
#include <stdio.h>

int main() {
    int total = 0;
    int n;
    
    while(1) {
        scanf("%d", &n);
        if(n == 5) {
            break;
        }
        
        switch(n) {
            case 1:
                total += 100;
                break;
            case 2:
                total += 120;
                break;
            case 3:
                total += 200;
                break;
            case 4:
                total += 60;
                break;
        }
    }
    
    printf("Bye Bye\n");
    printf("Total Calories: %d\n", total);
    
    return 0;
} 

File: TOI-Zero_68/A1/040/040.py
a = 0
while True:
    n = int(input())
    if n == 5:
        break
    if n == 1:
        a += 100
    elif n == 2:
        a += 120
    elif n == 3:
        a += 200
    elif n == 4:
        a += 60

print("Bye Bye")
print("Total Calories:", a)
